lib/templateFactory.js

```diff
@@ -287,14 +292,12 @@ class TemplateFactory extends BaseFactory {
             return Promise.all([
                 jobFactory.list(metricJobsListConfig),
                 buildFactory.list(metricBuildsListConfig),
-                jobFactory.list(jobListConfig)
+                jobFactory.query(jobsQueryConfig)
             ]).then(([jCount, bCount, jobs]) =>
                 templates.map(t => {
                     const jobCount = jCount.find(j => j.templateId === t.id);
-
                     const buildCount = bCount.find(b => b.templateId === t.id);
-                    const pipelineIds = jobs.filter(p => p.templateId === t.id).map(p => p.pipelineId);
-                    const pipelineCount = new Set(pipelineIds).size;
+                    const pipelineCount = jobs[0].find(p => p.templateId === t.id);
```

Comment:
We should name/rename the variable to reflect what it exactly represent.
```suggestion
                    const pipelineCount = pCount.find(p => p.templateId === t.id);
```